neqo-http3/src/features/extended_connect/webtransport_streams.rs

```diff
@@ -185,6 +185,14 @@ impl SendStream for WebTransportSendStream {
         }
     }
 
+    fn set_sendorder(&mut self, conn: &mut Connection, sendorder: Option<i64>) {
+	conn.stream_sendorder(self.stream_id, sendorder).ok();
```

Comment:
again, you are ignoring the return value here in a very dodgy fashion; .unwrap() if you mean it, but I might prefer that you return `Res<()>`.

Reply:
Modified to return Res<()> and in the places that implies as well.  I have to do the same map_err(|_| Error::InvalidStreamId)  as above because Error and neqo_transport::Error are different